module/mailer/src/Resources/config/services.yml

```diff
@@ -0,0 +1,25 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Ergonode\Mailer\Application\:
+        resource: '../../Application/*'
+
+    Ergonode\Mailer\Application\Command\:
+        resource: '../../Application/Command/*'
+        tags: [ 'console.command' ]
```

Comment:
It's part of autoconfiguration. Should not be needed.

Reply:
Yes and we should erase it from other modules. First module was created on SF 4.1. We must revalidate all :)